packages/product/integration-tests/__tests__/services/product-module-service/product-variants.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { TestDatabase } from "../../../utils"
+import { setupModuleService } from "../../../utils"
+import { ProductModuleService } from "@services"
+import { Product, ProductVariant } from "@models"
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+import { ProductTypes } from "@medusajs/types"
+
+describe("ProductModuleService product variants", () => {
```

Comment:
todo: I suggest to keep that in the module.spec.ts test in a separate describe eventually. what do you think? there is already some tests on the product

Reply:
That file will end up being super massive, I'd prefer to keep each separate.

Reply:
I see, I am fine with that I can clean what I ve done in the module tests. We can keep it that way and I ll clean up after at the same time I ll move the tests I have already done

Reply:
No, don't worry about it, i'll end up there anyway, will cleanup once I get to that point.